README.md

```diff
@@ -150,6 +150,8 @@ A companion ESLint plugin exists, [eslint-plugin-preact-i18n](https://www.npmjs.
     -   [Examples](#examples-4)
 -   [intl](#intl)
     -   [Parameters](#parameters-5)
+-   [defaultProvider](#defaultprovider)
```

Comment:
I think the term "provider" has been overloaded a lot and it could become confusing, can we call this the `pluralizer` instead?